src/ripple/app/paths/impl/PaySteps.cpp

```diff
@@ -423,6 +429,12 @@ toStrandV2 (
         if (hasCurrency && hasIssuer &&
             isXRP(pe.getCurrency()) != isXRP(pe.getIssuerID()))
             return {temBAD_PATH, Strand{}};
+
+        if (hasIssuer && (pe.getIssuerID() == noAccount()))
```

Comment:
Path elements may specify a combination of account, currency, and issuer. Specifying _just_ an issuer is valid, so
the current pe may not have a currency specified. Calling `pe.GetCurrency()` on such a pe would be a bug. If it does have it specified line 430 handles the check.